### PR TITLE
Fix scheduled container access mongo fail

### PR DESCRIPTION
**Pull request by RammusXu**
Forgot to add CRON_ENV.
```
Failed: error getting auth version of server: not authorized on admin to execute command { getParameter: 1, authSchemaVersion: 1 }
```

### CODE REVIEW

File: backup.sh [hunk 1]
```diff
@@ -29,13 +29,13 @@ if [[ $MONGO_DB_NAMES ]]; then
...
     do
         if [[ ! $BACKUP_FILE_NAME ]]; then
-            filename="$FILE-$d"
+            filename="$FILE-$d.gz"
         fi
         eval $command --archive=$filename -d $d $filter_errors
         echo "$(date +%Y-%m-%d:%H:%M:%S) dumped database: $filename"
     done
 else
-    eval $command --archive=$FILE $filter_errors
```
Comment (tenorok) [inline backup.sh:38]: Is `.gz` really necessary?

**Comment by tenorok** [comment]
Link previous pull: https://github.com/tenorok/docker-mongodump/pull/1

File: backup.sh (hunk 1 above)
Comment (RammusXu) [reply backup.sh:38]: It's not necessary.
But, when I want to decompress `.gz`, I use `gunzip FILE`.
It must have suffix.
```
rammus@Rammus-MacBook-Pro.local:~$ gunzip 20170621_071421
gunzip: 20170621_071421: unknown suffix -- ignored
```

And my colleague said it's more maintainable. So, I added it.

Comment (RammusXu) [inline backup.sh]:
```
rammus@Rammus-MacBook-Pro.local:~$ gunzip 20170621_071421
gunzip: 20170621_071421: unknown suffix -- ignored
```

**Pull request merged**